pyoutline/wrappers/opencue_wrap_frame

```diff
@@ -16,7 +16,7 @@ endif
 set dir=$1
 
 # Reset the value of TMPDIR
-setenv framedir /mcp/${CUE_JOB}/${CUE_FRAME}
+setenv framedir /tmp/${CUE_JOB}/${CUE_FRAME}
```

Comment:
We should probably make these configurable as environment variables. Although this can be done as a separate issue. Would you like me to create one?

Reply:
make's sense. please create a ticket